File: java/src/main/java/com/allstask/models/BillingTypeEnum.java
// This file is @generated
package com.allstask.models;

import com.fasterxml.jackson.annotation.JsonValue;
import com.allstask.Utils.ToQueryParam;

public enum BillingTypeEnum implements ToQueryParam {
    ADVANCE("ADVANCE"),
    ARREARS("ARREARS");
    private final String value;

    BillingTypeEnum(String value) {
        this.value = value;
    }

    @JsonValue
    public String getValue() {
        return this.value;
    }

    @Override
    public String toQueryParam() {
        return this.value;
    }
}

File: typescript/src/models/billingTypeEnum.ts
// this file is @generated
/* eslint @typescript-eslint/no-explicit-any: 0 */

export enum BillingTypeEnum {
    Advance = 'ADVANCE',
    Arrears = 'ARREARS',
    }

export const BillingTypeEnumSerializer = {
    _fromJsonObject(object: any): BillingTypeEnum {
        return object;
    },

    _toJsonObject(self: BillingTypeEnum): any {
        return self;
    }
}